README.md

```diff
@@ -26,6 +26,24 @@ This bundle can be installed following these steps:
         );
     }
 
+### Configure the bundle ###
+
+To enable the twitter connector set the consumer in the config
+
+    # app/config/config.yml
+    integrated_social:
+        twitter:
+            consumer_key: '%twitter_consumer_key%'
+            consumer_secret: '%twitter_consumer_secret%'
+
+To enable the facebook connector set the consumer in the config
+
+    # app/config/config.yml
+    integrated_social:
+        facebook:
+            app_id: '%twitter_app_id%'
+            app_secret: '%twitter_app_secret%'
+
```

Comment:
Can you update the docs for facebook, the values should be prefixed with facebook.